webpack/react_app/components/jobInvocations/AggregateStatus/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const AggregateStatus = statuses => (
```

Comment:
If you will migrate it to
```jsx
const AggregateStatus = ({ statuses }) => (...);
```

You would be able to do `{statuses.success}` instead `{statuses.statuses.success}`

Reply:
Fixed, that was goofy...